src/sections/hero.html

```diff
@@ -0,0 +1,47 @@
+<section
+  class="h-cg-s lg:h-cg-l lg:max-w-cg-l overflow-hidden flex lg:relative lg:items-end lg:mt-8 lg:mb-64 my-14 px-5"
+>
+  <div
+    class="lg:max-h-cg-l-herodiv1 max-w-cg-s lg:max-w-cg-l-herodiv1 absolute lg:top-36 lg:right-[43rem] min-h-cg-s-herodiv1 top-[15.5rem] right-11 md:max-h-cg-md-herodiv1 md:max-w-cg-md-herodiv1 md:right-[6.8rem] md:top-[19rem]"
+  >
+    <img class="h-full w-full object-cover" src="../../img/pink.png" alt="" />
```

Comment:
The `alt` is needed in case the image is not yet loaded so the user can see the name  of image.